src/Lexer.php

```diff
@@ -418,7 +418,7 @@ private function stripcslashes(string $str, string $quoteType): string
                 $result .= $nextChar;
             } elseif ("'" === $nextChar || '"' === $nextChar) {
                 if ($nextChar !== $quoteType) {
-                    trigger_deprecation('twig/twig', '3.12', 'Character "%s" at position %d in string on line %d should not be escaped; the "\" character is ignored in Twig v3 but will not be in v4. Please remove the extra "\" character.', $nextChar, $i + 1, $this->lineno);
+                    trigger_deprecation('twig/twig', '3.12', 'Character "%s" at position %d in string on line %d should not be escaped; the "\" character is ignored in Twig v3 but will not be in v4. Please remove the extra "\" character or add another one if you want to print the escape character too.', $nextChar, $i + 1, $this->lineno);
```

Comment:
I'm not sure we need this alternative, as the deprecation generally explains how to keep the same behavior.

Reply:
Alright, removed it.